pkg/tekton/metapipeline/metapipeline.go

```diff
@@ -35,6 +35,9 @@ const (
 	createTektonCRDsStepName = "create-tekton-crds"
 
 	tektonBaseDir = "/workspace"
+
+	mavenSettingsSecretName = "jenkins-maven-settings" // #nosec
```

Comment:
Is there really no other way? The meta pipeline should be agnostic of these things and it should be possible to configure this in a generic way. Having Maven/Nexus specific things in the meta-pipeline like this seems wrong.

Reply:
We already effectively do - the default `setVersion` for the maven build pack calls `mvn`, which is why we need `builder-maven` for the step image. It's janky, but it is what it is.

Reply:
This, fwiw, is why in the pre metapipeline days, we always had to have `jenkins-maven-settings` mounted in the `pipelinerunner`.

Reply:
I see.